### ISSUE TITLE
Network error when searching for external profile

### ISSUE THREAD

**Issue by mrstegeman** [nextcloud/social#346]
**Describe the bug**
When searching for a user on an external Mastodon instance, I received a 500 error from the server with the following content:

```json
{
  "status": -1,
  "exception": "OCA\\Social\\Exceptions\\InvalidResourceException",
  "message": "An exception occurred while executing 'INSERT INTO `oc_social_cache_documents` (`id`, `type`, `url`, `media_type`, `mime_type`, `error`, `local_copy`, `public`, `creation`) VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [\"https:\\\/\\\/cloud.domain.com\\\/documents\\\/g\\\/e4b42290-deb5-4591-a710-fb8b0d3b7766\", \"Image\", \"https:\\\/\\\/cf.mastohost.com\\\/v1\\\/AUTH_91eb37814936490c95da7b85993cc2ff\\\/socialsamsunginternet\\\/accounts\\\/avatars\\\/000\\\/000\\\/138\\\/original\\\/7dede47f4e13c352.jpg\", \"image\\\/jpeg\", \"\", 0, \"\", \"0\", \"2019-01-17 15:46:41\"]:\n\nSQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'url' at row 1"
}
```

**To Reproduce**
1. Go to Social app
2. Search for `@rzr@social.samsunginter.net`
3. View error in browser dev tools

**Expected behavior**
I should see the user's profile.

**Client details:**
 - OS: macOS 10.14
 - Browser: Firefox
 - Version: Nightly (66.0a1)
 - Device: Desktop

<details>
<summary>Server details</summary>

**Social app version:** 0.1.2

**Operating system**: Official Docker image

**Web server:** Apache

**Database:** mariadb 10.3.11

**PHP version:** 7.2

**Nextcloud version:** 15.0.2

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
{"reqId":"vuCAhoIKfnvBdK1bEdch","level":1,"time":"2019-01-17T15:52:22+00:00","remoteAddr":"172.20.0.6","user":"michael","app":"social","method":"GET","url":"/apps/social/api/v1/global/account/info?account=@rzr@social.samsunginter.net","message":"[>>] request: {\"host\":\"social.samsunginter.net\",\"url\":\"\\/.well-known\\/webfinger\",\"timeout\":10,\"type\":0,\"data\":{\"resource\":\"acct:rzr@social.samsunginter.net\"}} - result: {\"subject\":\"acct:rzr@social.samsunginter.net\",\"aliases\":[\"https://social.samsunginter.net/@rzr\",\"https://social.samsunginter.net/users/rzr\"],\"links\":[{\"rel\":\"http://webfinger.net/rel/profile-page\",\"type\":\"text/html\",\"href\":\"https://social.samsunginter.net/@rzr\"},{\"rel\":\"http://schemas.google.com/g/2010#updates-from\",\"type\":\"application/atom+xml\",\"href\":\"https://social.samsunginter.net/users/rzr.atom\"},{\"rel\":\"self\",\"type\":\"application/activity+json\",\"href\":\"https://social.samsunginter.net/users/rzr\"},{\"rel\":\"salmon\",\"href\":\"https://social.samsunginter.net/api/salmon/138\"},{\"rel\":\"magic-public-key\",\"href\":\"data:application/magic-public-key,RSA.qtd1FwHd3-hArnHLV6nwlcjgF51Hepb-vaHPhXhhvOntLu8V0c4JXYQOojX1GAiGxQPJfBWKpA6GO7c38vHFyQ2MqvfvgGTo43HRNaipTztrjG1OMzvETfq_Y_CM1Srj7WXlDvUGk3SPS5EO57q-iYe_AVSElIyGeI5TMTEjWOqe7swolhTBjj7uuyEKS-WD9eb4ecadfzaeV9oxX2atl6NwrOIYW5pzdQLfWTrC6HtallY_4etEJ5ZrmH07GI_iJLxoujAZsm627WECPrjQUOw-wareffQsivJT9VTI-3IKNznYFrqdaOqd5hwNKULcE4pNdFOLRAbzzKceETaa8Q==.AQAB\"},{\"rel\":\"http://ostatus.org/schema/1.0/subscribe\",\"template\":\"https://social.samsunginter.net/authorize_interaction?uri={uri}\"}]}","userAgent":"Mozilla/5.0 (Windows NT 6.1; rv:60.0) Gecko/20100101 Firefox/60.0","version":"15.0.2.0","id":"5c40a4b72fe5f"}
```

#### Browser log
No console errors, just the 500 error response from the server.

</details>

**Comment by juliushaertl** [comment]
@daita 

> String data, right truncated: 1406 Data too long for column 'url' at row 1

**Comment by daita** [comment]
This should be fixed in 0.2.0: https://github.com/nextcloud/social/pull/352/files#diff-f34870c53ff0583f4d67d2ea5d71587aR58

**Comment by mrstegeman** [comment]
This seems fixed now. Thanks!
